tests/PHPStan/Analyser/ScopePhpVersionTest.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Analyser;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Exit_;
+use PHPStan\Testing\TypeInferenceTestCase;
+use PHPStan\Type\VerbosityLevel;
+
+class ScopePhpVersionTest extends TypeInferenceTestCase
+{
+
+	public function dataTestPhpVersion(): array
+	{
+		return [
+			[
+				'int<80000, 80499>',
+				__DIR__ . '/data/global-scope-constants.php',
```

Comment:
And a test for a namespaced file would be nice

Reply:
here you are